content/cloud-computing/concepts/cloud-services/cloud-services.md

```diff
@@ -0,0 +1,80 @@
+
+- [Infrastructure as a service (IaaS)](https://www.codecademy.com/resources/docs/cloud-computing/iaas)
+- [Platform as a service (PaaS)](https://www.codecademy.com/resources/docs/cloud-computing/paas)
+- [Function as a service (FaaS)](https://www.codecademy.com/resources/docs/cloud-computing/faas)
+- [Software as a service (SaaS)](https://www.codecademy.com/resources/docs/cloud-computing/saas)
+
+## AWS
+
+AWS is the largest cloud computing platform, and the most mature, introducing its first public cloud services in 2006. It serves millions of users, from individuals to large enterprises, to governments.
+
+- AWS offers more than 200 services to its customers.
+- As of 2021 AWS has 25 geographic regions and 218+ edge locations.
+
+The advantages of AWS include:
+
+- The widest array of services; from networking to quantum computing.
+- The most mature, reliable and secure platform.
+- The most compute capability.
+
+Disadvantages include:
+
+- If Dev/Enterprise support is needed, users must pay for it separately.
+- The number of services and options can be overwhelming.
+- A complex and hard-to-understand pricing structure.
+- There are limited options to run a [hybrid cloud](https://www.codecademy.com/resources/docs/cloud-computing/types-of-cloud) solution.
+
+## Azure
+
+Azure is the second largest cloud platform, introduced my Microsoft in 2008. It is also currently among the fastest growing cloud platforms.
+
+- Azure offers more than 200 products and services, a wide array of which are tailored to enterprises already using Microsoft products.
+- In addition to Microsoft products, Azure supports [open-source](https://www.codecademy.com/resources/docs/open-source) languages, technologies and platforms.
+- As of 2021 Azure has 60+ geographic regions with 116 edge locations.
+
+The advantages of Azure include:
+
+- The ease of integration and migration for existing Microsoft users.
+- Lower cost for users of Microsoft products such as Windows, Active Directory and SQL Server.
+- Significant support for hybrid cloud solutions.
+- The broadest geographic reach of the three platforms.
+
+Disadvantages include:
+
+- Fewer service offerings than AWS.
+- Predominantly supports enterprise customers.
+- High level of expertise and maintenance required.
+
+## GCP
+
+GCP was introduced by Google in 2010 and is the smallest of the big three cloud-computing platforms.
+
+- GCP offers over 100 services including computing, networking and [big data](https://www.codecademy.com/resources/docs/general/big-data).
+- GCP services include Google Workspace, enterprise Android, and Chrome OS.
+- As of 2021 GCP has 27 geographic regions and 146 edge locations.
```

Comment:
What does "edge location" mean?

Reply:
They're data centers close to the users, designed to deliver services with the lowest latency possible

Reply:
Gotcha!